pkg/apis/componentconfig/v1alpha1/conversion.go

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
what prevented us from generating these conversions?

Reply:
@mtaufen i saw some warnings in zz_generated.conversion.go. Are those the things that prevented us from fully auto generating? 

```
zz_generated.conversion.go:	// WARNING: in.LeaderElection requires manual conversion: inconvertible types (*k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1.LeaderElectionConfiguration vs k8s.io/kubernetes/pkg/apis/componentconfig.LeaderElectionConfiguration)
zz_generated.conversion.go:	// WARNING: in.VolumeConfiguration requires manual conversion: inconvertible types (*k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1.VolumeConfiguration vs k8s.io/kubernetes/pkg/apis/componentconfig.VolumeConfiguration)
zz_generated.conversion.go:	// WARNING: in.LeaderElection requires manual conversion: inconvertible types (k8s.io/kubernetes/pkg/apis/componentconfig.LeaderElectionConfiguration vs *k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1.LeaderElectionConfiguration)
zz_generated.conversion.go:	// WARNING: in.VolumeConfiguration requires manual conversion: inconvertible types (k8s.io/kubernetes/pkg/apis/componentconfig.VolumeConfiguration vs *k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1.VolumeConfiguration)
zz_generated.conversion.go:	// WARNING: in.PersistentVolumeRecyclerConfiguration requires manual conversion: inconvertible types (*k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1.PersistentVolumeRecyclerConfiguration vs k8s.io/kubernetes/pkg/apis/componentconfig.PersistentVolumeRecyclerConfiguration)
zz_generated.conversion.go:	// WARNING: in.PersistentVolumeRecyclerConfiguration requires manual conversion: inconvertible types (k8s.io/kubernetes/pkg/apis/componentconfig.PersistentVolumeRecyclerConfiguration vs *k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1.PersistentVolumeRecyclerConfiguration)
```

Reply:
Do the substructures need to be pointers in the v1alpha1 type? If not I think you can avoid manual conversions.

Reply:
Ack. Let me try that @mtaufen